gateway/gateway.go

```diff
@@ -162,6 +162,7 @@ type HandleT struct {
 	addToBatchRequestQWaitTime                                 stats.RudderStats
 	trackSuccessCount                                          int
 	trackFailureCount                                          int
+	activeClientCount                                          int
```

Comment:
use `int32`

Reply:
making it int32, so that we can do atomic addition & subtraction.